frontend/packages/shared/src/components/GiteaHeader/VersionControlButtons/components/ShareChangesPopover/CommitAndPushContent/FileChangesInfoModal/FileChangesInfoModal.tsx

```diff
@@ -58,9 +58,7 @@ export const FileChangesInfoModal = ({
         );
       case 'error':
         return (
-          <Alert size='small' severity='danger'>
```

Comment:
Are you sure that the `size='small'` should be removed? Ref. previous comments, we should be able to keep the prop small after we exposing the `Alert` API. 😊

Reply:
Agree 😊, I will set it back